src/utils/LookmlDiagrammer/diagrammer.ts

```diff
@@ -161,17 +166,17 @@ export function generateExploreDiagram(explore: ILookmlModelExplore, hiddenToggl
     return joinPath
   })
 
-  // General order tables would be arranged in, if no joins and no base view
+  // General order tables would be arranged in, if no base view
   const joinCount = countJoins(diagramDict)
   let buildOrder = views.sort((a: string, b: string)=>{
     return joinCount[a] > joinCount[b] ? -1 : 1;
   })
 
   const diagrammer = new LookmlDiagrammer(diagramDict, buildOrder, explore)
   
-  const baseViewName = diagramDict.tableData[explore.name] ? explore.name : buildOrder[0]
+  const seedName = baseViewName !== "" ? baseViewName : buildOrder[0]
```

Comment:
Is seedName the best name here? What's it used for?

Reply:
Will change. seedName is the table name to start arranging the diagram from